CSS/Task2/style.css

```diff
@@ -0,0 +1,105 @@
+html {
+    font-family: 'Poppins', arial, sans-serif;
+    font-size: 14px;
+}
+
+@font-face {
+    font-family: 'Poppins';
```

Comment:
In Figma you can find that the name of font in table is **Inter**. Download it and add to the project